app/workers-a.fly.toml

```diff
@@ -0,0 +1,26 @@
+# See https://fly.io/docs/reference/configuration/ for information about how to use this file.
+
+app = 'metapage-workers-a'
+primary_region = 'iad'
+
+[build]
+#  image = 'metapage/metaframe-docker-worker:0.25.0'
```

Comment:
I removed region information, and replaced the build lines with the line referring to the published `image` directly since we'll be publishing versioned images and that'll be the primary way they're referenced across platforms, I imagine. For now I set them to the `latest-standalone` tag that doesn't exist yet, but they can be switched to a pinned version number later if desired.